superset-frontend/plugins/plugin-chart-word-cloud/src/legacyPlugin/transformProps.ts

```diff
@@ -34,6 +34,16 @@ function getMetricLabel(
   return metric.label;
 }
 
+function getSeriesLabel(
+  series: LegacyWordCloudFormData['series'],
+): string | undefined {
+  if (typeof series === 'string' || typeof series === 'undefined') {
+    return series;
+  }
+  
+  return series.label;
+}
+
```

Comment:
I just did a very similar fix yesterday to the Heatmap plugin. Please check the `getColumnLabel` util and how it can be used here: #23507

Reply:
Thanks for pointing me to the util function! I have made the changes.